### DISCUSSION TITLE
If I want to create a backup of the data to be deleted

### DISCUSSION THREAD

**Discussion by wuyanfeiwork** [minio/minio#18749] [Q&A] [answered]
I'm planning to delete data uploaded one year ago, but I'd like to create a backup of this data before deletion. How can I proceed?

It seems that I can only perform a full backup of the bucket and there is no support for selectively backing up data from a specific number of days ago.

**Comment by harshavardhana** [comment, marked as answer]
@wuyanfeiwork for architecture discussions, reach out to us at https://min.io/pricing

**Reply by wuyanfeiwork** [reply]
@harshavardhana  Is it not possible to specify data from xxx days ago for backup and then delete it in the open-source version? Can this requirement be fulfilled in the enterprise edition?

**Reply by klauspost** [reply]
@wuyanfeiwork You can reach out on `hello@min.io` with questions.
